docs/source/connecting/connecting.md

```diff
@@ -34,6 +34,52 @@ async fn main() -> Result<(), Box<dyn Error>> {
 After successfully connecting to some specified node the driver will fetch topology information about
 other nodes in this cluster and connect to them as well.
 
+Scylla Serverless is an elastic and dynamic deployment model. When creating a `Session` you need to
+specify the secure connection bundle as follows:
+
+```rust
+# extern crate scylla;
+# extern crate tokio;
+use std::path::Path;
+use std::error::Error;
+use scylla::CloudSessionBuilder;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let session = CloudSessionBuilder::new(Path::new("config_data.yaml"))
+        .unwrap()
+        .build()
+        .await
+        .unwrap();
+
+    Ok(())
+}
+
```

Comment:
Nit: unnecessary empty line at the end of the snippet (the previous example doesn't have it).